content/learn/keyconcepts.adoc

```diff
@@ -0,0 +1,164 @@
+
+Key characteristics of values files include:
+
+* Plain YAML Format: The human-readable and easy-to-edit syntax of YAML makes configuration settings accessible and straightforward to manage.
+* Hierarchical Nature: Values files support a hierarchy of values, allowing logical organization and structuring of configurations, which is especially useful in handling complex deployments.
+In the context of Helm charts, values files are crucial components that define configuration settings for deploying applications and managing resources within an OpenShift Container Platform cluster. They enable flexible, per-cluster customization while ensuring consistency with the overall validated pattern. This ensures that organizations can achieve efficient, secure, and consistent deployments across multiple OpenShift Container Platform clusters.
+
+A common practice is to use a base values file, such as `values-global.yaml`, for global settings, and then have cluster-specific values files for example `values-cluster1.yaml`, `values-cluster2.yaml` that override or add to the global settings. This approach allows for comprehensive customization while maintaining a centralized and organized configuration structure, promoting best practices for deployment and resource management.
+
+For more information, see link:https://validatedpatterns.io/learn/values-files/[Values].
+
+[id="applications"]
+== Applications
+
+The applications section in the Helm values file plays a crucial role in defining and managing the deployment of various applications within an OpenShift Container Platform cluster. By leveraging Helm charts and adhering to validated patterns, it ensures consistency, best practices, and simplified management, leading to reliable and scalable application deployments.
+
+The path field in each application entry points in the values file points to the location of the Helm chart and associated configuration files. These charts contain the Kubernetes manifests and configuration necessary to deploy the application. Helm charts are used to package Kubernetes applications and manage their deployment in a consistent and reproducible manner.
+
+When these applications are deployed, the following Kubernetes resources are typically created:
+
+* *Deployments*: Define the desired state and replicas for the application's pods.
+* *Services*: Expose the application's pods to other services or external traffic.
+* *ConfigMaps and Secrets*: Store configuration data and sensitive information.
+* *PersistentVolumeClaims (PVCs)*: Request storage resources for the application.
+* *Ingress or Routes*: Provide external access to the application.
+* *RBAC (Role-Based Access Control)*: Define access permissions and roles.
+
+[id="RHCM"]
+== Red Hat Advanced Cluster Management (RHCM) 
+
```

Comment:
I would maybe add here that RHACM is only installed when a pattern spans multiple clusters